src/Illuminate/Mail/Transport/ArrayTransport.php

```diff
@@ -49,7 +49,7 @@ public function messages()
     /**
      * Clear all of the messages from the local collection.
      *
-     * @return void
+     * @return Collection
```

Comment:
Please don't make unrelated changes in the PR. This change also doesn't match our code standards anyway.